recipes/kiwisolver/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.1.3" %}
+package:
+  name: kiwisolver
+  version: {{ version }}
+
+source:
+  fn: kiwisolver-{{ version }}.tar.gz
+  url: https://github.com/nucleic/kiwi/archive/{{ version }}.tar.gz
+  sha256: 761757e93f884223281af7969df85a93542a66769ca13617084baf82848c656d
+  # patches:
+  #   - setup.py.patch
```

Comment:
@ericdill is this good to go or are you adding a patch here? (If not let's remove the comments and merge as-is)